pkg/gcp/machine_controller_util.go

```diff
@@ -122,6 +123,14 @@ func (ms *MachinePlugin) CreateMachineUtil(ctx context.Context, machineName stri
 				},
 			}
 		}
+		if disk.KmsKeyName != "" {
```

Comment:
is there a possibility that someone provides only spaces as the keyName? Safe thing to do would be to just trim `disk.KmsKeyName`